lib/pages/slider_selector.dart

```diff
@@ -0,0 +1,298 @@
+class RangeSliderData {
+  double min;
+  double max;
+  double lowerValue;
+  double upperValue;
+  int divisions;
+  bool showValueIndicator;
+  int valueIndicatorMaxDecimals;
+  bool forceValueIndicator;
+  Color overlayColor;
+  Color activeTrackColor;
+  Color inactiveTrackColor;
+  Color thumbColor;
+  Color valueIndicatorColor;
+  Color activeTickMarkColor;
+
+  static const Color defaultActiveTrackColor = const Color(0xFF0175c2);
+  static const Color defaultInactiveTrackColor = const Color(0x3d0175c2);
+  static const Color defaultActiveTickMarkColor = const Color(0x8a0175c2);
+  static const Color defaultThumbColor = const Color(0xFF0175c2);
+  static const Color defaultValueIndicatorColor = const Color(0xFF0175c2);
+  static const Color defaultOverlayColor = const Color(0x290175c2);
+
+  RangeSliderData({
+    this.min,
+    this.max,
+    this.lowerValue,
+    this.upperValue,
+    this.divisions,
+    this.showValueIndicator: true,
+    this.valueIndicatorMaxDecimals: 1,
+    this.forceValueIndicator: false,
+    this.overlayColor: defaultOverlayColor,
+    this.activeTrackColor: defaultActiveTrackColor,
+    this.inactiveTrackColor: defaultInactiveTrackColor,
+    this.thumbColor: defaultThumbColor,
+    this.valueIndicatorColor: defaultValueIndicatorColor,
+    this.activeTickMarkColor: defaultActiveTickMarkColor,
+  });
+
+  // Returns the values in text format, with the number
+  // of decimals, limited to the valueIndicatedMaxDecimals
+  //
+  String get lowerValueText =>
+      lowerValue.toStringAsFixed(valueIndicatorMaxDecimals);
+  String get upperValueText =>
+      upperValue.toStringAsFixed(valueIndicatorMaxDecimals);
+
+  // Builds a RangeSlider and customizes the theme
+  // based on parameters
+  //
+  Widget build(BuildContext context, frs.RangeSliderCallback callback) {
+    return Container(
+      width: double.infinity,
+      child: Row(
+        children: <Widget>[
+          Container(
+            constraints: BoxConstraints(
+              minWidth: 40.0,
+              maxWidth: 40.0,
+            ),
+            child: Text(lowerValueText),
+          ),
+          Expanded(
+            child: SliderTheme(
+              // Customization of the SliderTheme
+              // based on individual definitions
+              // (see rangeSliders in _RangeSliderSampleState)
+              data: SliderTheme.of(context).copyWith(
+                overlayColor: overlayColor,
+                activeTickMarkColor: activeTickMarkColor,
+                activeTrackColor: activeTrackColor,
+                inactiveTrackColor: inactiveTrackColor,
+                //trackHeight: 8.0,
+                thumbColor: thumbColor,
+                valueIndicatorColor: valueIndicatorColor,
+                showValueIndicator: showValueIndicator
+                    ? ShowValueIndicator.always
+                    : ShowValueIndicator.onlyForDiscrete,
+              ),
+              child: frs.RangeSlider(
```

Comment:
I see this `frs.RangeSlider` multiple times. Find every occurrence and extract them to a method.